pygfx/renderers/wgpu/meshshader.py

```diff
@@ -221,7 +267,7 @@ def code_vertex(self):
             // set with the public API and we assume that it does not include a flip.
             let winding_world = get_sign_of_det_of_4x4(world_transform);
             let winding_cam = get_sign_of_det_of_4x4(u_stdinfo.cam_transform);
-            sub_index = select(sub_index, -1 * (sub_index - 1) + 1, winding_world * winding_cam < 0.0);
+            //sub_index = select(sub_index, -1 * (sub_index - 1) + 1, winding_world * winding_cam < 0.0);
```

Comment:
This was not meant to be removed, I think?
```suggestion
            sub_index = select(sub_index, -1 * (sub_index - 1) + 1, winding_world * winding_cam < 0.0);
```

Reply:
yep forgot to uncomment it sorry!